scripts/src.build.quick.sh

```diff
@@ -16,8 +16,9 @@ fi
 
 LD_FLAGS="-s -w -X github.com/docker-slim/docker-slim/pkg/version.appVersionTag=${TAG} -X github.com/docker-slim/docker-slim/pkg/version.appVersionRev=${REVISION} -X github.com/docker-slim/docker-slim/pkg/version.appVersionTime=${BUILD_TIME}"
 
-mkdir -p ${BDIR}/dist_linux/
-rm -f ${BDIR}/dist_linux/*
+BINDIR="${BDIR}/bin"
+mkdir -p "$BINDIR"
+rm -f "${BINDIR}/"*
 
-CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="${LD_FLAGS}" -mod=vendor -o "${BDIR}/dist_linux/docker-slim" "${BDIR}/cmd/docker-slim/main.go"
-CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="${LD_FLAGS}" -mod=vendor -o "${BDIR}/dist_linux/docker-slim-sensor" "${BDIR}/cmd/docker-slim-sensor/main.go"
+CGO_ENABLED=0 go build -ldflags="${LD_FLAGS}" -mod=vendor -o "${BINDIR}/docker-slim" "${BDIR}/cmd/docker-slim/main.go"
+CGO_ENABLED=0 go build -ldflags="${LD_FLAGS}" -mod=vendor -o "${BINDIR}/docker-slim-sensor" "${BDIR}/cmd/docker-slim-sensor/main.go"
```

Comment:
what about GOOS=linux GOARCH=amd64

Reply:
Must there be a platform restriction for a dev build intended for local use? Do certain platform targets not work?

Reply:
The sensor app is always linux (arch can be arm potentially). The main app can be anything (no restrictions)

Reply:
Oh right, specifically for the sensor. I’ll add those variables back. In fact I’m going to move these script changes to a separate PR since they’re unrelated to the probe feature.

Reply:
not a big deal... same is fine